.rspec

```diff
@@ -1 +1 @@
---require spec_helper
+--require rails_helper
```

Comment:
I would have created this commit in a separate pull request, because it doesn't feel related to this story.
No need to change it now, but might be worth considering for incoming PRs

Reply:
@pmanrubia good point, will do this next time.